source/index.txt

```diff
@@ -95,11 +100,33 @@ What's New
 The :doc:`What's New </whats-new>` section lists new features and
 changes in each version.
 
-Additional Learning
--------------------
+Resources
+---------
+
+Visit the Developer Hub and MongoDB University for resources related to
+the MongoDB Java driver.
+
+Developer Hub
+~~~~~~~~~~~~~
+
+The Developer Hub contains content, resources, community, and social
+engagement for developers.
+
+To view articles on Java, see the `How To's and Articles page
+<https://www.mongodb.com/developer/learn/?content=Articles&text=Java>`_.
+
+To view the forum on Java, see the `forums page
+<https://www.mongodb.com/community/forums/tag/java>`_.
+
+MongoDB University
+~~~~~~~~~~~~~~~~~~
+
+MongoDB University contains free courses to inspire and educate everyone
+to learn MongoDB. It has the best-in-class training and empowers
```

Comment:
Let's avoid opinionated descriptions like "inspire" and "best-in-class". Readers may lose confidence/trust when seeing opinionated material.